addons/TBMod_persistence/functions/fn_modulepersistencezeus.sqf

```diff
@@ -7,57 +7,51 @@ params ["_logic", "", "_activated"];
 if !(local _logic) exitWith {true};
 deleteVehicle _logic;
 if !(_activated) exitWith {true};
-    
-0 spawn { 
- 
-private _dialogResult = 
-[ 
-    "Persistence", 
-    [ 
-        ["Operation", ["Speichern","Laden"], 0] 
-    ] 
-] call Ares_fnc_showChooseDialog; 
-if (_dialogResult isEqualTo []) exitWith{systemChat "[TBMod_persistence] Abbruch"}; 
- 
-_dialogResult params ["_operation"]; 
- 
-private _save = if (_operation == 0) then {true} else {false}; 
- 
-if(_save) then { 
-    private _dialogResult2 = 
-    [ 
-        "Persistence", 
-        [ 
-            ["Save Name", "", ""] 
-        ] 
-    ] call Ares_fnc_showChooseDialog; 
- 
-    if (_dialogResult2 isEqualTo []) exitWith{systemChat "[TBMod_persistence] Abbruch"}; 
- 
-    _dialogResult2 params ["_name"]; 
-    [_save, _name] call TB_fnc_persistence; 
-    if(profileNamespace getVariable ["TB_persistence_savednames",[]] isEqualTo []) then { 
-        profileNamespace setVariable ["TB_persistence_savednames",[_name]]; 
-    } 
-    else 
-    { 
-        profileNamespace getVariable ["TB_persistence_savednames",[]] pushBackUnique _name; 
-    } 
- 
-}else
-    { 
-        private _saves = profileNamespace getVariable ["TB_persistence_savednames",[]]; 
-        private _dialogResult2 = 
-        [ 
-            "Persistence", 
-            [ 
-                ["Save Name", _saves, 0] 
-            ] 
-        ] call Ares_fnc_showChooseDialog; 
- 
-        if (_dialogResult2 isEqualTo []) exitWith{}; 
- 
-        _dialogResult2 params ["_nameidx"]; 
-        [_save, _saves select _nameidx] remoteExec ["TB_fnc_persistence", 2]; 
+
+[] spawn
+{
+    private _dialogResult = ["Persistence", [
+            ["Operation", ["Speichern", "Laden"], 0],
+            ["Ort", ["Server", "Lokal"], 0],
+        ]] call Ares_fnc_showChooseDialog;
+    if (_dialogResult isEqualTo []) exitWith {systemChat "[TBMod_persistence] Abbruch"};
+    _dialogResult params ["_operation", "_ort"];
+
+    private _save = _operation == 0;
+    private _server = _ort == 0;
+    private _saves = if (_server) then
+    {
+        [profileNamespace, "TB_persistence_savedNames", []] spawn BIS_fnc_getServerVariable;
```

Comment:
Warum mit spawn? Wir sind doch bereits in ner scheduled Umgebung

Reply:
HAHAHAHAHAHA 😄 
Schau mal was ich vor einer halben Stunde om Wiki geändert habe -> https://community.bistudio.com/wiki/BIS_fnc_getServerVariable

Hatte es vergessen im Code zu ändern, guter Spot ^^

Reply:
Deswegen hab ichs geändert